subsys/dfu/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Kconfig - DFU support configuration options
```

Comment:
It's either that or "lib". I expect this will grow over time (especially once we start querying mcuboot capabilities etc. at runtime) into something that looks more like a subsystem than a library, so I agree.